modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AspNetCoreServerCodegen.java

```diff
@@ -482,4 +513,61 @@ private void setOperationIsAsync() {
             additionalProperties.put(OPERATION_IS_ASYNC, operationIsAsync);
         }
     }
+
+    private void setIsFramework() {
+        if (aspnetCoreVersion.getOptValue().startsWith("3.")) {
+            // default, do nothing
```

Comment:
this block is setting a field and has a side effect of assigning a key/value in `additionalProperties`. Saying "do nothing" could be confusing. Similarly, "3.x" isn't the default defined in this type and could be confusing. Should probably remove comment since the log message below provides runtime and in-code explanation.